app/views/layouts/application.html.erb

```diff
@@ -80,10 +80,10 @@
   <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/leaflet-search@2.4.0/dist/leaflet-search.min.css">
   </head>
 
-  <body>
```

Comment:
I don't think you are supposed to remove body tag. What do you think?

Reply:
I've added the body tag in the `loading_page` partial.. so I thought I should remove this one.. should I add it back?

Reply:
Yes! for sure. It's the view of our main page. See, partials just get rendered at places where you put the render tag. You get better understanding from - https://guides.rubyonrails.org/layouts_and_rendering.html .

Reply:
It didn't change the results of my tests when I added the <body> back, but I saw that _loading_page.html.erb has another set of body tags inside it which I don't think should be there? I just don't know how to relate that back to the error messages

Reply:
I also think here body tag might not be the reason as there is no if condition or so and DOM would eventually get complete with body tag in partial. I am checking if something else is wrong

Reply:
Right now through my latest commit, I've removed the body tags in the partial and added the body tags in the html code..

Reply:
Looking into it!